ui/src/components/EdgePinCluster.tsx

```diff
@@ -0,0 +1,127 @@
+import * as React from "react";
+import L, { LatLng } from "leaflet";
+import "react-leaflet-markercluster/dist/styles.min.css";
+import "leaflet.markercluster";
+import MarkerClusterGroup from "react-leaflet-markercluster";
+import "leaflet/dist/leaflet.css";
+import { EdgePin, getAnchor } from "./EdgePin";
+import { EdgeType } from "./EdgePins";
+import { arrow } from "./Arrow";
+
+const createClusterCustomIcon = function (edgeType: EdgeType) {
```

Comment:
Prefer arrow function in this case.

```suggestion
const createClusterCustomIcon = (edgeType: EdgeType) => {
```